package.json

```diff
@@ -14,6 +14,7 @@
     "build:live": "cp envConfig/live.env .env && npm run webpack:prod",
     "cypress:proxy": "export NO_PROXY=localhost,localhost.bbc.com && cypress",
     "cypress": "npm run cypress:proxy -- run",
+    "cypress:ci": "npm run cypress:proxy -- run | perl -pe 's/\\x1b\\[[0-9;]*m//g'",
```

Comment:
what the hell is perl doing in here?

Reply:
(Discussed offline, repeating here for visibility) Strips out all ANSI colour codes, making cypress output in jenkins logs legible